test/controllers/patients_controller_test.rb

```diff
@@ -1047,5 +1047,88 @@ def teardown; end
     h = History.where(patient: patient)
     assert_match(/changed Symptom Onset Date/, h.second.comment)
   end
+
+  test 'setting and clearing follow up flag for an individual patient with no other household members' do
+    user = create(:public_health_enroller_user)
+    sign_in user
+    patient = create(:patient, creator: user, monitoring: true, continuous_exposure: true)
+
+    post :update_follow_up_flag, params: {
+      id: patient.id,
+      follow_up_reason: 'Hospitalized',
+      follow_up_note: 'Test Note',
+      clear_flag: false,
+      clear_flag_reason: '',
+      apply_to_household: false,
+      apply_to_household_ids: []
+    }, as: :json
+
+    assert_response :success
+    patient.reload
+    assert_match('Hospitalized', patient.follow_up_reason)
+    assert_match('Test Note', patient.follow_up_note)
+    assert_match('Flagged for Follow-up. Reason: "Hospitalized: Test Note"', History.where(patient: patient, created_by: user.email)[0].comment)
```

Comment:
There are recently added test_case helpers for asserting that a patient has a history that contains a specific comment
https://github.com/SaraAlert/SaraAlert/blob/master/test/test_case.rb#L17